src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+}
+
+.body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+}
```

Comment:
you need to reset the padding from the left side

<img width="385" alt="image" src="https://user-images.githubusercontent.com/55802630/172862128-14530dd3-3c2e-4ece-8a22-8607274a2f65.png">